frontend/__tests__/components/Flex/flex.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render, screen } from "../../../test-utils";
+import Flex from "../../../components/Flex/Flex";
+
+describe("Flex", () => {
+  it("should render flex component", () => {
+    render(<Flex>big flex</Flex>);
+    expect(screen.getByText("big flex")).toBeInTheDocument();
```

Comment:
You are missing tests here. Please re-examine some testing best practices, make sure you are covering all of the scenarios and behaviors you should expect from your component.